install_scripts/templates/kubernetes-init.sh

```diff
@@ -27,6 +27,7 @@ BOOTSTRAP_TOKEN_TTL="24h"
 KUBERNETES_NAMESPACE="default"
 KUBERNETES_VERSION="{{ kubernetes_version }}"
 NO_CE_ON_EE="{{ no_ce_on_ee }}"
+HARD_FAIL_ON_LOOPBACK="{{ hard_fail_on_loopback }}"
```

Comment:
This is either `None` or `True`.  How is each stringified? Most importantly is, `None` rendered as an empty string?

Reply:
`None` is rendered as an empty string. `True` is rendered as the string "True".